week-5/F-webchat/exercise-2.js

```diff
@@ -12,18 +12,36 @@ send the message to store it on the remote server. Use the following API:
 HTTP Verb: POST
 API: https://codeyourfuture.herokuapp.com/api/messages
 Request Body: { "content": "some text" }
-
-
-===============
-Expected result
-===============
-
-After opening index.html in your browser, write a message in the input field and click
-on the submit button. Then check the following:
-
-1) The input field should be empty.
-2) When you refresh the page in your browser, you should be able to see your new message in the message list.
 */
 
-
-// Write your code here
+document.getElementById("submit").addEventListener("click", () => {
+  var requestBody = {
+    content: document.getElementById("message-input").value
```

Comment:
```suggestion
var input = document.querySelector("#message-input");
 var requestBody = { "content": value };
```

Reply:
value is undefiend in your request body ,,,,i guess .
if like this one is possible i think =>
  var input = document.querySelector("#message-input").value;
 var requestBody = { "content": value };